handbook/engineering/hiring/software-engineer-frontend.md

```diff
@@ -2,15 +2,22 @@
 
 We are looking for frontend engineers who know how to build intuitive user experiences to make the power of [Universal Code Search](https://about.sourcegraph.com/product) accessible to everyone. You will have a lot of ownership to solve tough technical and UX problems in key areas of our web application and browser extensions (e.g. code search, code navigation, campaigns).
 
+The following engineering teams are currently hiring for this role:
+
+1. [Web](../web/index.md#growth-plan) - all experience levels
+2. [Cloud](../cloud/index.md#growth-plan) - senior experience level
+
 ## About you
 
 - You have practice at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g. cut scope) to ship quickly and iterate when necessary.
+- You are proficiency in the English language, both written and verbal, sufficient for success in a remote and largely asynchronous work environment.
 - You communicate clearly and empathetically, especially in writing and documentation.
-- You are [high agency](../../../company/values.md#high-agency).
+- You share our [values](../../../company/values.md), and work in accordance with those values
+- You have the ability to thrive in an all-remote and largely asynchronous organization
 
 ## Your responsibilities
 
-- Work closely with a dedicated designer and PM to implement high-quality UIs.
+- Work closely with a dedicated designer and PM to implement high-quality UIs using React, SCSS and TypeScript.
```

Comment:
@felixfbecker I call this out specifically here as I make mention of it as a requirement for the coding excercies. Keen to hear your thoughts on this and whether we should expand it with the framework we use for testing for instance.